docs/notebooks/inequality_constraints.pct.py

```diff
@@ -146,6 +148,94 @@ def create_bo_model(data):
 )
 plt.show()
 
+
+# %% [markdown]
+# ## Define a batch acquisition rule
```

Comment:
i prefer the title you used in the other notebook

> Batch-sequential strategy

what do you reckon?

Reply:
happy to change - in the long term we might need to sort out the doc and have a descriptive, distinct title for each subsection, but that's not part of this PR